rfd/0169-auto-updates-linux-agents.md

```diff
@@ -479,9 +536,12 @@ However, any changes to `agent_schedules` that occur while a group is active wil
 Releasing new agent versions multiple times a week has the potential to starve dependent groups from updates.
 
 Note that the `default` schedule applies to agents that do not specify a group name.
+[TODO: It seems we removed the default bool, So we have a mandatory default group? Can we pick the last one instead?]
```

Comment:
We got feedback that the boolean was confusing during the feedback sessions. No strong opinion. How about last-as-default, but only if no default group is explicitly defined?